golang/gorm/README.md

```diff
@@ -0,0 +1,28 @@
+# Compile, Deploy and Run
+Install required components
+```
+go get github.com/google/uuid
+go get github.com/gorilla/mux
```

Comment:
"github.com/jinzhu/gorm" seems to be a requirement as well

Reply:
think gorm is basic requirement just like golang itself, rather than a dependency. but added that anyway.